content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -190,6 +192,14 @@ sudo apt-mark hold kubelet kubeadm kubectl
 ```
 {{% /tab %}}
 {{% tab name="CentOS, RHEL or Fedora" %}}
+Please note that Kubernetes provides all official `rpm` packages via
+[yum.k8s.io](https://yum.k8s.io).
+{{< warning >}}
+The `kubernetes-cni` package was obsoleted with the release of Kubernetes
+v1.18.4, v1.17.7 and v1.16.11 on June 17th, 2020. This means in case of
+doing an upgrade from prior versions, please ensure that the package gets
+**manually** removed by executing `yum remove kubernetes-cni-0.7.5-0`.
```

Comment:
Is it needed/worth mentioning when the package should be removed (e.g. should it be removed before upgrading or after)?

Reply:
Yes, I added it as a note. Thank you!